### DISCUSSION TITLE
How to publish null event message to Kafka compact topic for tombstone?

### DISCUSSION THREAD

**Discussion by cookie-bytes** [BEagle1984/silverback#187] [General]
I am trying to publish a null event message to Kafka compact topic so removed from the topic.
However, Silverback is checking the event is not and appears to need the event to work out the relevant routing.
Is there a way to publish a null event message which I am missing?

**Comment by BEagle1984** [comment]
Yes, see the documentation here: https://silverback-messaging.net/concepts/broker/tombstone.html#producer.
